packages/database/src/migrations/20210222204800-UpdateEntityDhisIdToTrackEntityInstanceId-modifies-data.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`
+    -- We want to drop the entity trigger because we just want to change the id key -> trackedEntityId, no actual data is changed, this shouldn't trigger any change listener (eg: for pushing)
+    DROP TRIGGER IF EXISTS entity_trigger
+    ON entity;
```

Comment:
Same like above, just want to update the key `dhis.id` to `dhis.trackedEntityId`, no need to fire change listeners